capture/exception.js

```diff
@@ -1,5 +1,5 @@
 import Raven from 'raven'
-import Config from '../config'
+import Config from '../config/index.js'
```

Comment:
why don't use `import Config from '../config/' ? because it has file index?

Reply:
expectations don't match reality :disappointed: 
![image](https://user-images.githubusercontent.com/41193120/134617414-7e5a4f13-e486-4764-b7ce-d0ce36dee5a9.png)